types/zapier-platform-core/index.d.ts

```diff
@@ -0,0 +1,114 @@
+// Type definitions for zapier-platform-core 3.1
+// Project: https://github.com/zapier/zapier-platform-core
+// Definitions by: Bradley Ayers <https://github.com/bradleyayers>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.4
```

Comment:
It can be Typescript 2.5 here.

Reply:
Does 2.4 imply it *doesn't* work with 2.5? I thought this field was the "oldest" compatible version of TypeScript?

Reply:
Maybe I'm wrong, is there any document about this? I didn't find any.

Reply:
Yes, see https://github.com/DefinitelyTyped/DefinitelyTyped#i-want-to-use-features-from-typescript-21-or-above